### ISSUE TITLE
Please tell me how to edit and compile AI.

### ISSUE THREAD

**Issue by ajujoa** [jncraton/BWMetaAI#39]
I want to edit ai can you teach me how to do it? (Windows 10, Starcraft-remastered)
I wish the default environment settings were the detailed way.

**Comment by jncraton** [comment]
The basics of building the project from source are described here:

https://github.com/jncraton/BWMetaAI/blob/master/CONTRIBUTING.md

You'd then need to add/modify your own builds (.pyai files in the source directory).

If you haven't done any BW modding before, you might want to start with [PyMS](https://github.com/poiuyqwert/PyMS). It provides the tools for editing the various game assets that this project modifies. [PyAI](https://github.com/poiuyqwert/PyMS/blob/master/Help/Programs/PyAI.md) in particular might be useful to you if you want to create custom scripts.

**Comment by ajujoa** [comment]
Thank you. 
However, I still fails to create the bin file.
1. Ubuntu
2. install nodejs
3. altinstall python 2.7.18
4. install git
5. clone github
6. 'make bins', 'make -f makefile' not work.

![2023-06-26 01 00 47](https://github.com/jncraton/BWMetaAI/assets/96732137/e6da6162-b6ec-43c1-a62c-6f8f696a0f35)

I'm not sure about is _'This particular version is required to be able to interact with storm.dll using ctypes.'_

I am a Windows user and have never used Ubuntu. You may be doing it now through Googling and not understanding it.

**Comment by jncraton** [comment]
It looks like you don't have python2 available. The underlying tool used here still requires Python 2, but Python 2 is no longer maintained and may be difficult to install on your system. I haven't used this recently, so I'm not sure how to best proceed.

**Comment by duran55** [comment]
I installed python 2 successfully on Ubuntu 20.04. You need these packages:
apt install python2:i386 python-tk:i386

**Comment by ajujoa** [comment]
I succeeded.
Thank you very much.
Now take a step.
--------------------------------------------------------------------------
Install Ubuntu 20.04 (Not latest version!!)
Update basic tools before installation: sudo apt update
Install build-related tools: sudo apt install -y build-essential
Install Node.js: sudo apt install nodejs
Installed python 2: apt install python2:i386 python-tk:i386
Install Git: sudo apt install git
Finally, clone the BWMetaAI repository using the following command
 :git clone https://github.com/jncraton/BWMetaAI.git

Move to 'BWMetaAI' folder.
Mouse R --> 'Open in Terminal'
make <-- command!!
--------------------------------------------------------------------------
I succeeded compile.
